Documentation/gettingstarted/cli-download.rst

```diff
@@ -8,30 +8,25 @@ various features (e.g. clustermesh, Hubble).
     .. code-block:: shell-session
 
       CILIUM_CLI_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/cilium-cli/master/stable.txt)
-      curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-linux-amd64.tar.gz{,.sha256sum}
-      sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
-      sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
-      rm cilium-linux-amd64.tar.gz{,.sha256sum}
+      CLI_ARCH=$(arch)
```

Comment:
It seems not all distributions include the `arch` command in their coreutils package. For example I don't have it on Arch Linux (oh the irony). Could we use `uname -m` instead for Linux?

Reply:
Yeah Robin mentioned this as well. I try using `uname` for everything